[LLM] chore: Rename GEMINI.md to AGENTS.md and update instructions (#16)
AnySoftKeyboard/janus: AGENTS.md

- Before creating a commit, always run `./gradlew testDebugUnitTest` to ensure all tests are passing and `./gradlew spotlessApply` to ensure proper formatting.

## Commit Messages
- **Title:** The title should be concise and prefixed with `[LLM]`.
- **Description:** Include a short description of the issue (bug, feature-request, crash, chore, etc) and a short description of the solution.

> - Instruct the Agent to add its name at the end of the description to signify the commit was made by an AI Agent.

Author: ```suggestion
- Add your name at the end of the description to signify the commit was made by an AI Agent.
```

> - Instruct the Agent to add its name at the end of the description to signify the commit was made by an AI Agent.

Author: ```suggestion
- Add your name at the end of the description to signify the commit was made by an AI Agent.
```
